DesignSafe-CI/portal
# Task/DES-1862 - Refactor project management modal and post methods.

## Overview: ##
This is a refactor of the Manage Project modal. The Manage Project model needed to be reworked in order to accomplish the following:

1. We need to be able to create and update specific fields for amend/revise (without having to send back the complete project model).
2. We need to prevent users from changing their project data once it's been published (This is only going to be allowed in the amend/revise pipeline).
3. Fix several UI bugs.

Still working to add a "Version / Amend" model which will display a form with limited inputs.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1862](https://jira.tacc.utexas.edu/browse/DES-1862)

## Testing Steps: ##
1. Load up any project (especially "other") and make changes using the new modal. It should be in the same place as before
2. Add and remove users, fields, dates...
